docs/topics/signals.txt

```diff
@@ -121,7 +121,7 @@ Here's how you connect with the decorator::
     from django.core.signals import request_finished
     from django.dispatch import receiver
 
-    @receiver(request_finished)
+    @receiver(request_finished, **kwargs)
```

Comment:
`kwargs` should be documented in the signature above:
```
.. function:: receiver(signal, **kwargs)
```
and in the list of params:
```
:param **kwargs: ...
```

Reply:
@felixxm the PR looks good now?